NuRadioMC/EvtGen/NuRadioProposal.py

```diff
@@ -248,7 +238,11 @@ def __init__(self, config_file='SouthPole', log_level=logging.INFO, tables_path=
             upper_energy_limit of tables that will be created by PROPOSAL, in NuRadioMC units (eV).
             There will be an error if primaries with energies above this energy will be injected.
             Note that PROPOSAL will have to regenerate tables for a new values of upper_energy_limit
-
+        create_new: bool (default:False)
+            Can be used to force the creation of a new ProposalFunctions object.
+            By default, the __init__ will only create a new object if none already exists.
+            For more details, check the documentation for the
+            `Singleton metaclass <NuRadioReco.utilities.html#NuRadioReco.utilities.metaclasses.Singleton>`_.
```

Comment:
This doesn't work. To cross-reference a class, you need to use the `:class:` reference (see [the documentation](https://nu-radio.github.io/NuRadioMC/Introduction/pages/contributing.html#links-and-cross-references)), which also doesn't have a trailing underscore. (rst is a pain, don't blame me!)
Alternatively, to have  a 'normal' hyperlink, you need to link to the actual webpage (which starts with https://...), at the risk that this will break at some point.

Reply:
should be fixed